stripe/src/test/java/com/stripe/android/StripeTest.java

```diff
@@ -248,6 +253,204 @@ public void createBankAccountTokenSynchronous_withValidBankAccount_returnsToken(
     }
 
     @Test
+    public void createSourceSynchronous_withBitcoinParams_passesIntegrationTest() {
+        Stripe stripe = new Stripe(mContext);
+        SourceParams bitcoinParams = SourceParams.createBitcoinParams(1000L, "usd", "abc@def.com");
```

Comment:
I have the ability to do it - wasn't sure how it was yet used, so I haven't added testing to support it yet. I was thinking it would be a public setter on the SourceParams object, or perhaps an optional map that you can pass in.